ui/src/translations/en.json

```diff
@@ -108,5 +108,8 @@
   "volume_statue_failed_pv_bound_hint": "The volume is bound to a storage request.",
   "delete_a_volume": "Delete a volume",
   "delete_a_volume_warning": "Deleting this volume will permanently delete the data it contains.",
-  "delete_a_volume_confirm": "Do you want to delete "
+  "delete_a_volume_confirm": "Do you want to delete ",
+  "select_a_storageClass": "Select a Storage Class...",
```

Comment:
Please use `…` instead of `...` (apply to the string below and to the French translation).